webui-page/webui.js

```diff
@@ -220,8 +231,13 @@ window.onkeydown = function(e) {
     },
   ];
   for (var i = 0; i < bindings.length; i++) {
-    if (e.keyCode === bindings[i].code || e.key === bindings[i].key) {
-      send(bindings[i].command, bindings[i].param1, bindings[i].param2);
+    const binding = bindings[i];
+    if (e.keyCode === binding.code || e.key === binding.key) {
+      if (typeof binding.command === 'string') {
+        send(binding.command, binding.param1, binding.param2);
+      } else {
+        binding.command();
+      }
```

Comment:
I'd like to have this refactored as follows:

For all existing keybindings, command should be set to the function `send`. Then we can use `param1` and `param2` for the mpv command and the (optional) parameter. It'd make sense to also rename `command` to `func` or similar.

So this:

``` js
    {
      "key": "]",
      "code": 221,
      "command": "speed_adjust",
      "param1": "1.1",
    },
```

would then become this:

``` js
    {
      "key": "]",
      "code": 221,
      "func": send,
      "param1": "speed_adjust",
      "param2": "1.1",
    },
```

Then we can just do:

``` js
    if (e.keyCode === binding.code || e.key === binding.key) {
        binding.func(binding.param1, binding.param2);
      }
```

Reply:
i was thinking this would get refactored to not have "param" at all.

```js
    {
      "key": "]",
      "code": 221,
      "func": () => send("speed_adjust", "1.1"),
    },
```

wdyt ?

Reply:
Awesome! I'm not very JS savvy :)

Reply:
that's ok ... i'm terrible at lua